Automattic/jetpack
# Remove all CodeClimate references

#### Changes proposed in this Pull Request:

* Remove CodeClimate

#### Jetpack product discussion

p9dueE-2ND-p2 

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Check for "CodeClimate" mentions in the codebase
* Make sure that CodeClimate is pending in checks list
